shared/trainer/defaults.yaml

```diff
@@ -124,14 +124,16 @@ dis:
     init_type: kaiming
     init_gain: 0.2
     use_sigmoid: false
-    num_D: 1 #Number of discriminators to use (>1 means multi-scale)
+    num_D: 3 #Number of discriminators to use (>1 means multi-scale)
```

Comment:
why 3 again ?

Reply:
Yeah, I actually think keeping 1 as default is better as well

Reply:
Victor suggested 3 Lol. Okay, I will change it back to 1.